### ISSUE TITLE
SatisMeter is already shown

### ISSUE THREAD

**Issue by pykaso** [satismeter/satismeter-ios#17]
Hi, I'm getting "SatisMeter is already shown" from our app when I trying to force show a Survey after a user closed it sometime in the past. 

I can't find any SatisMeter related views in views hierarchy. So I assume it's really closed.

Of course, this happens only when application stays in memory. After the new app start it's ok.

**Comment by jsedlacek** [comment]
@pykaso thanks for the bug report. Let us have a look why this happens.

Just for the context: What are you trying to achieve with the `forceSurvey` flag? Is that for testing purposes, or are you trying to implement the survey in certain scenarios?

**Comment by pykaso** [comment]
Yes, now it's for testing purposes during integrating SatisMeter into our app. I want to be sure, that this issue doesn't prevent opening Satismeter in production.

**Comment by jsedlacek** [comment]
@pykaso Thanks. It should not matter in production, as it's unlikely that you'd like to show the survey multiple times within one application session.

We will address this in future releases.

**Comment by pykaso** [comment]
The same issue also occurs without a `forceSurvey` tag.  When the user of the application (same device) has changed. It means calling the `identifyUser` with another `userID` value.

**Comment by jsedlacek** [comment]
Thanks for an update on this. This is obviously a bug, although very rare case.

I will keep the issue open until we get to resolve it.
